static/app/components/charts/areaChart.tsx

```diff
@@ -25,7 +25,7 @@ export function transformToAreaSeries({
     AreaSeries({
       stack: stacked ? 'area' : undefined,
       name: seriesName,
-      data: data.map(({name, value}) => [name, value]),
+      data: data?.map(({name, value}) => [name, value]),
```

Comment:
would be nice to have a test

Reply:
Agreed, would have to figure a good place for it, there's no tests for this component. So maybe somewhere higher up? 
Any objections to doing this in another PR to unblock?

Reply:
let's do it in another PR!